app/src/main/java/org/mozilla/fenix/home/recentbookmarks/view/RecentBookmarksHeaderViewHolder.kt

```diff
@@ -48,7 +48,7 @@ class RecentBookmarksHeaderViewHolder(
     @Composable
     override fun Content() {
         Column {
-            Spacer(modifier = Modifier.height(40.dp))
+            Spacer(modifier = Modifier.height(34.dp))
```

Comment:
Where are the values for these Spacer changes coming from?

Reply:
Because in the changes of this PR the tap area for the `Show all` button is increased, I decreased the space around the header where the `Show all` button is used, to maintain the previous aspect.

Reply:
I haven't had a chance to pull down the code and play around with it. My secondary concern is that these values might be overcompensating since I changed the header to center vertically in https://github.com/mozilla-mobile/fenix/pull/26963/files when we bumped up the Header font size.